partner_middlename/README.rst

```diff
@@ -0,0 +1,119 @@
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+You can configure some common name patterns for the inverse function
+in Settings > Configuration > General settings:
+
+* Lastname SecondLastname Firstname: For example 'Anderson Lavarge Robert'
+* Lastname SecondLastname, Firstname: For example 'Anderson Lavarge, Robert'
+* Firstname Lastname SecondLastname: For example 'Robert Anderson Lavarge'
+
+After applying the changes, you can recalculate all partners name clicking
+"Recalculate names" button. Note: This process could take so much time depending
+how many partners there are in database.
+
+You can use *_get_inverse_name* method to get firstname, lastname and
+second lastname from a simple string and also *_get_computed_name* to get a
+name form the firstname, lastname and second lastname.
+These methods can be overridden to change the format specified above.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Edit any partner's form.
+* Make sure the partner is not a company.
+* Enter firstname and lastnames.
+
+If you directly enter the full name instead of entering the other fields
+separately (maybe from other form), this module will try to guess the best
+match for your input and split it between firstname, lastname and second
+lastname using an inverse function.
+
+If you can, always enter it manually please. Automatic guessing could fail for
+you easily in some corner cases.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/partner-contact/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/partner-contact/issues/new?body=module:%20partner_second_lastname%0Aversion:%2016.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Tecnativa
+
+Contributors
+~~~~~~~~~~~~
+
+* Jairo Llopis <jairo.llopis@tecnativa.com>.
+* Antonio Espinosa.
+* Pedro M. Baeza <pedro.baeza@tecnativa.com>.
+* Nikul Chaudhary <nikulchaudhary2112@gmail.com>
+* Manuel Regidor <manuel.regidor@sygel.es>
+
+Maintainers
+~~~~~~~~~~~
+
+This module is maintained by the OCA.
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/partner-contact <https://github.com/OCA/partner-contact/tree/16.0/partner_second_lastname>`_ project on GitHub.
+
+You are welcome to contribute. To learn how please visit https://odoo-community.org/page/Contribute.
```

Comment:
Part of this are still from the copied module you used as a starting point.
In that case it is better to replace the full content with a line such as "AUTOMATICALLY GENERATED".

Reply:
@dreispt Yes code updated